WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -184,6 +184,11 @@ - (void)updateExistingEndUser:(NSInteger)endUserID {
     NSMutableURLRequest *urlRequest = [self requestWithURL:url HTTPMethod:@"PUT" andHTTPBody:nil];
 
     NSURLSessionDataTask *dataTask = [_wootricSession dataTaskWithRequest:urlRequest completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+      if (error || data == nil) {
```

Comment:
Is it necessary to check that `data == nil`? I would think we just need to see if there's an error and report on that?